docs/api.md

```diff
@@ -284,55 +324,31 @@ All entities derived from this instance (e.g. processes, the file system, etc.)
 
 ---
 
-## `isPreviewMessage`
```

Comment:
I'm remove from the docs this function (although it's still exported) as it's no longer necessary to call it in userland. The `@webcontainer/api` package in the latest version calls it automatically for our end users so that they don't have to worry about forward compatibility.